content/applications/services/helpdesk/advanced/track_and_bill.rst

```diff
@@ -141,16 +146,16 @@ On the ticket detail form, click on the :guilabel:`Timesheets` tab and click :gu
 Choose an :guilabel:`Employee`, add a :guilabel:`Description` of the task, and enter the number of
 :guilabel:`Hours Spent`.
 
-As new lines are added to :guilabel:`Timesheets` tab, the :guilabel:`Remaining Hours on SO` field is
-automatically updated.
+As new lines are added to :guilabel:`Timesheets` tab, the :guilabel:`Remaining Hours on SO` field,
+at the bottom-right of the tab, is automatically updated.
 
 .. image:: track_and_bill/track-bill-remaining-hours-total.png
    :align: center
    :alt: View of the timesheets tab on a ticket with an emphasis on the remaining hours on an SO.
 
 .. note::
    If the number of hours on the :guilabel:`Timesheets` tab exceeds the number of hours sold, the
-   :guilabel:`Remaining Hours of SO` will turn red.
+   :guilabel:`Remaining Hours of SO` turns red.
 
 As hours are added to the :guilabel:`Timesheets` tab, they are automatically updated in the
 :guilabel:`Delivered` field on the :abbr:`SO (sales order)`, as well.
```

Comment:
For the *Invoice post-paid support services* section, please update to use italics since there isnt an instruction directing the user to the service product where the "Invoicing Policy" field is. 
Also, may as well add the reference to the section. :)
(I cant add a suggestion to the correct area because it was not modified in this PR. Lines 167 to 169)
```suggestion
Invoice post-paid support services
==================================

When support services are billed based on the amount of time spent on an issue, an invoice cannot be
created before the total number of hours required to solve the problem have been entered on a
timesheet. In this case, a service product with the *Invoicing Policy* set to *Based on Timesheets*
would be used, like the :ref:`the section above <helpdesk/configure-service-products>`.
```